uberpoet/resources/Info.plist

```diff
@@ -5,11 +5,11 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>Uber Mock App$(DISPLAY_NAME_SUFFIX)</string>
```

Comment:
those were never set and the buck build was failing. I figured we can just remove them.